Content.Shared/Ghost/Roles/GhostRolesEuiMessages.cs

```diff
@@ -10,6 +10,14 @@ public struct GhostRoleInfo
         public string Name { get; set; }
         public string Description { get; set; }
         public string Rules { get; set; }
+        public EntityUid Owner { get; set; }
+
+        public string UserId { get; set; }
+
+        public override string ToString()
+        {
+            return (UserId != "") ? $"{Identifier}: {Name} ({UserId})" :  $"{Identifier}: {Name}";
+        }
```

Comment:
not sure i fully understand what the point of this is?

Reply:
ToString makes debugging much nicer because its used in the watch window

Reply:
i mean the userid stuff with ghost roles, not the tostring specifically